spec/lib/miq_automation_engine/miq_ae_service_model_spec.rb

```diff
@@ -77,6 +77,18 @@ module MiqAeServiceModelSpec
         expect(@ae_vm.tag_unassign("#{category.name}/non_exisiting_tag")).to be_truthy
       end
     end
+
+    describe "#taggable" do
+      it "Vm model is taggable" do
+        expect(@ae_vm.taggable?).to be_truthy
+      end
+
+      it "Lan model is not taggable" do
+        lan = FactoryGirl.create(:lan)
+        lan_provider = MiqAeMethodService::MiqAeServiceLan.new(lan.id)
+        expect(lan_provider.taggable?).to be_falsey
+      end
+    end
```

Comment:
@pkomanek Please add a test to verify you get the proper exception when calling a tagging method on an model that does not support tagging.